src/components/MoneyRequestConfirmationList.js

```diff
@@ -500,7 +503,7 @@ function MoneyRequestConfirmationList(props) {
         }
 
         const shouldShowSettlementButton = props.iouType === CONST.IOU.TYPE.SEND;
-        const shouldDisableButton = selectedParticipants.length === 0;
+        const shouldDisableButton = selectedParticipants.length === 0 || shouldDisplayMerchantError;
```

Comment:
Coming from https://github.com/Expensify/App/issues/38130:

Form Submit buttons should not be disabled or blocked from being pressed